dev/com.ibm.ws.jaxws.ejb/bnd.bnd

```diff
@@ -47,13 +47,12 @@ Service-Component: \
      implementation:=com.ibm.ws.jaxws.ejb.EJBHandlerResolver; \
      provide:='com.ibm.wsspi.ejbcontainer.WSEJBHandlerResolver'; \
      configuration-policy:=optional; \
-     properties:="service.vendor=IBM", \
+     properties:="service.vendor=IBM"
 
 Export-Package: \
     com.ibm.ws.jaxws.ejb
 
 Import-Package: \
-   javax.xml.bind.*;version="[2.2,3)", \
```

Comment:
I'm pretty sure these imports were needed for Java 11 compatibility. Before you deliver these changes, be sure to do a Java 11 personal build.

Same comment applies to all other `javax.xml.bind` removals

Reply:
I have compared the bundle manifest files for all of open liberty image from builds before my change and after my changes.  The Import and Export package statements are identical.  Only the Private packages have differences.  The private packages have differences for packages that I removed that don't exist or have a changed order and are the same.